README.md

```diff
@@ -22,11 +22,9 @@ Keyvi - the short form for "Key value index" - defines a special subtype of the
 
 ### Quick
 
-Precompiled binaries for Ubuntu 14.04LTS are available at the [keyvi PPA](https://launchpad.net/~hendrik-muhs/+archive/ubuntu/keyvi). To install use: 
```

Comment:
I would like to keep this installation method. But having that said, I think PyPi should be the 1st mentioned install method. 

I can move 'Other installation options' into a separate file and link it.

Reason: Not everybody prefers PyPi and future it might be interesting to install only the C++ part.